config/main.py

```diff
@@ -142,6 +143,14 @@ def read_json_file(fileName):
         raise Exception(str(e))
     return result
 
+# write given JSON file
+def write_json_file(json_input, fileName):
+    try:
+        with open(fileName, 'w') as f:
+            json.dump(json_input, f, indent=4)
+    except Exception as e:
+        raise Exception(str(e))
```

Comment:
except and raise immediately does not add value. It is the same as no except at all.